pkts-sip/src/test/java/io/pkts/packet/sip/address/impl/TelURIImplTest.java

```diff
@@ -0,0 +1,38 @@
+package io.pkts.packet.sip.address.impl;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.pkts.buffer.Buffer;
+import io.pkts.buffer.Buffers;
+import io.pkts.packet.sip.SipParseException;
+import io.pkts.packet.sip.address.TelURI;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+
+public class TelURIImplTest {

```

Comment:
Could you add some more tests around bad tel-uris?

Also, could you add some tests where you still have stuff left in the buffer and ensure whatever content that is left is indeed still there. I.e. 

```
Buffer buffer = Buffers.wrap("tel:+34600100100\r\nsome_other_stuff")
TelURI telUri = TelURI.frame(buffer);
assertEquals("34600100100", telUri.getPhoneNumber().toString());
assertTrue(telUri.isGlobal());
assertThat(buffer.toString(), is("some_other_stuff");```